### PR TITLE
feat(core): remove api registerOp and combine it with registerBackend and add README

### PR DESCRIPTION
**Pull request by JingyuanZhang**
… and add README

### CODE REVIEW

File: packages/paddlejs-core/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+[中文](./README_cn.md)
+
+# Paddle.js Core
+A core part of the Paddle.js ecosystem, this repo hosts `@paddlejs/paddlejs-core`,
```
Comment (zhongkai) [inline packages/paddlejs-core/README.md:4]: A -> The
`As the core part of the ....`
`repo` should be `module` or `package`, cause this repo hosts several packages.

File: packages/paddlejs-core/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+and provides interfaces for backend registration and environment variable registration.
+
+## Importing
+You can install this package via npm., `@paddlejs/paddlejs-core`
+
+```js
+import { registerBackend, Runner } from '@paddlejs/paddlejs-core';
+
+const runner = new Runner({
+    modelPath: '/model/mobilenetv2', // model path, eg http://xx.cc/path, http://xx.cc/path/model.json, /localModelDir/model.json, /localModelDir
```
Comment (zhongkai) [inline packages/paddlejs-core/README.md:15]: e.g.

File: packages/paddlejs-core/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    feedShape: {  // input shape
+        fw: 256,
+        fh: 256
+    },
+    fetchShape: [1, 1, 1920, 10],  // output shape
+    fill?: '#fff',   // fill color when resize image, default value is #fff
+    inputType?: 'image' // whether is image or video, default value is image
+});
+
+// You need to register backend and ops before initing runner
```
Comment (zhongkai) [inline packages/paddlejs-core/README.md:26]: initing -> initializing

File: packages/paddlejs-core/README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+registerBackend(
+    'webgpu', // 'webgl', 'webgpu', you can name it yourself
```
Comment (zhongkai) [inline packages/paddlejs-core/README.md:28]: by yourself

File: packages/paddlejs-core/README_cn.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
+[English](./README.md)
+
+# Paddle.js Core
+
+是 Paddle.js 推理引擎的核心部分，npm 包名是 `@paddlejs/paddlejs-core`，负责整个引擎的推理流程运行，提供计算方案注册、环境变量注册的接口。
+
+## Importing
```
Comment (zhongkai) [inline packages/paddlejs-core/README_cn.md:7]: 用`安装`？

File: packages/paddlejs-core/README.md (hunk 1 above)
Comment (JingyuanZhang) [reply packages/paddlejs-core/README.md]: get

**Pull request merged**